README.md

```diff
@@ -1,2 +1,207 @@
-# hotel-bookings-backend
-This is a Hotel bookings app that allows users to search a list of rooms availables and make reservations. This is the backend repo.
+<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍